workspaces/nitro-host/Makefile

```diff
@@ -10,10 +10,12 @@
 # licensing and copyright information.
 
 .PHONY: all build clean clean-cargo-lock clippy default doc fmt \
-	test-client test-dependencies test-server veracruz-test
+	test-client test-dependencies test-server veracruz-test build-tests
```

Comment:
Nit: "tests" is missing here?

Reply:
I will fix that in another PR, there are some other minor changes I have in mind.